charts/rasa-x/values.yaml

```diff
@@ -12,8 +12,12 @@ rasax:
   name: "rasa/rasa-x"  # gcr.io/rasa-platform/rasa-x-ee
   # tag refers to the Rasa X image tag (uses `appVersion` by default)
   tag: ""
-  # port on which Rasa X runs
-  port: 5002
+  # service
+  service:
+    # port on which Rasa X runs
+    port: 5002
+    # service type
+    type: ClusterIP
   # scheme by which Rasa X is accessible
```

Comment:
You miss `rasax.port` that is used in the rasa-x-service template

Reply:
Thanks, dropped this